### PR TITLE
nuScenes tutorials in Google Colab

### PR DESCRIPTION
**Pull request by whyekit-aptiv**
### This PR will:
- Update the nuScenes tutorials so that they can be run in Colab, if the user desires; as an example: https://colab.research.google.com/github/nutonomy/nuscenes-devkit/blob/tutorial_colab/python-sdk/tutorials/nuscenes_tutorial.ipynb

### CODE REVIEW

File: python-sdk/tutorials/nuscenes_lidarseg_tutorial.ipynb [hunk 1]
```diff
@@ -11,6 +11,32 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# !mkdir -p /data/sets/nuscenes  # Make the directory to store the nuScenes dataset in.\n",
+    "\n",
+    "# !wget https://www.nuscenes.org/data/v1.0-mini.tgz  # Download the nuScenes mini split.\n",
+    "# !wget https://www.nuscenes.org/data/nuScenes-lidarseg-mini-v1.0.tar.bz2  # Download the nuScenes-lidarseg mini split.\n",
```
Comment (holger-nutonomy) [inline python-sdk/tutorials/nuscenes_lidarseg_tutorial.ipynb:32]: @Qiang-Xu Is it okay to share the files like this?

File: python-sdk/tutorials/nuscenes_lidarseg_tutorial.ipynb [hunk 2]
```diff
@@ -11,6 +11,32 @@
...
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Google Colab (optional) \n",
+    "[![Google Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/nutonomy/nuscenes-devkit/blob/master/python-sdk/tutorials/nuscenes_lidarseg_tutorial.ipynb)\n",
+    "If you are running this notebook in Google Colab, you can uncomment the below cell and run it; everything will be set up nicely for you within Google Colab. Otherwise, go to [**Setup**](#Setup) to manually set up everything."
+   ]
```
Comment (holger-nutonomy) [inline python-sdk/tutorials/nuscenes_lidarseg_tutorial.ipynb:21]: Move section below "Setup"

File: python-sdk/tutorials/nuscenes_lidarseg_tutorial.ipynb [hunk 3]
```diff
@@ -11,6 +11,32 @@
     "This demo assumes that nuScenes is installed at `/data/sets/nuscenes`. The mini version (i.e. v1.0-mini) of the full dataset will be used for this demo."
    ]
   },
+  {
```
Comment (holger-nutonomy) [inline python-sdk/tutorials/nuscenes_lidarseg_tutorial.ipynb:14]: Add the same thing for nuImages and nuScenes tutorials.

File: python-sdk/tutorials/nuscenes_tutorial.ipynb [hunk 4]
```diff
@@ -35,6 +35,30 @@
     "![](https://www.nuscenes.org/public/images/nuscenes-schema.svg)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Google Colab (optional)\n",
+    "[![Google Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/nutonomy/nuscenes-devkit/blob/master/python-sdk/tutorials/nuscenes_tutorial.ipynb)\n",
```
Comment (holger-nutonomy) [inline python-sdk/tutorials/nuscenes_tutorial.ipynb:43]: logo
The Colab logo looks centered. Can you try to left-align it?

Comment (whyekit-aptiv) [reply python-sdk/tutorials/nuscenes_tutorial.ipynb]: Done

**Review by holger-nutonomy — APPROVED** [review]

**Comment by oscar-nutonomy** [comment]
Very neat. Thanks @whyekit-aptiv !

**Review by oscar-nutonomy — APPROVED** [review]

**Pull request merged**